plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -424,48 +393,6 @@ function CoverMenu:onHistoryMenuHold(item)
     UIManager:clearRenderStack()
 
     -- Add some new buttons to original buttons set
-    table.insert(orig_buttons, {
-        { -- Allow user to view real size cover in ImageViewer
-            text = _("View full size cover"),
```

Comment:
This explicite `View full size cover` is now replaced by a simpler `Book cover`, but I guess that's ok.